cli/context/context.go

```diff
@@ -163,3 +164,27 @@ type ConnectCtx struct {
 	Username string
 	Password string
 }
+
+type FailoverCtx struct {
+	File string
+	Mode string
+
+	StateProvider   string
+	FailoverTimeout int
+	FencingEnabled  bool
+	FencingTimeout  int
+	FencingPause    int
+
+	StateboardParams struct {
+		URI      string
```

Comment:
I think that we can not validate the params dictionary. It will allow supporting new failover parameters without patching CLI. I mean, just make `StateboardParams` and `Etcd2Params` of type `map[string]interface{}`. There is no need to validate these parameters, just pass them to Cartridge and it will check them.